context/app/routes_main.py

```diff
@@ -44,11 +44,7 @@ def details(type, uuid):
 
     entity = client.get_entity(uuid)
     entity_schema = load_yaml(open(current_app.root_path + '/schemas/entity.yml'))
```

Comment:
maybe the `entity.yml` should be a variable set at the top of this file.  also, maybe a `with open(...)`

Reply:
Added `with open`. Tried moving it up and out, but then `current_app` isn't available. Not sure about the best approach, but just added a TODO for now.

Reply:
ok - i meant just the `'/schemas/entity.yml'` as a variable since the `current_app` shouldn't change.